pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[tool.poetry]
```

Comment:
I was using requirements.txt because it worked well inside of Docker. If this is supposed to replace the requirements file then make sure this works when building the Docker container.

Reply:
It is possible to generate a requirements.txt from poetry and use it, or use poetry directly on docker

Reply:
Also, I haven't removed the requirements. poetry would work well for local development and some testing and running thinks locally.